presto-docs/src/main/sphinx/optimizer/history-based-optimization.rst

```diff
@@ -24,16 +24,17 @@ How to use HBO
-=========================================================== =========================================================================================================================================================================================================== ===============
+=========================================================== =========================================================================================================================================================================================================== ===================================
 use_history_based_plan_statistics                           Enable using historical statistics for query optimization                                                                                                                                                   False
 track_history_based_plan_statistics                         Enable recording the statistics of the current query as history statistics so as to be used by future queries                                                                                               False
 track_history_stats_from_failed_queries                     Track history based plan statistics from complete plan fragments in failed queries                                                                                                                          True
 history_based_optimizer_timeout_limit                       Timeout for history based optimizer                                                                                                                                                                         10 seconds
 restrict_history_based_optimization_to_complex_query        Enable history based optimization only for complex queries, i.e. queries with join and aggregation                                                                                                          True
-history_input_table_statistics_matching_threshold           When the size difference between current table and history table exceed this threshold, do not match history statistics. When value is 0, use the default value set by hbo.history-matching-threshold       0
-=========================================================== =========================================================================================================================================================================================================== ===============
+history_input_table_statistics_matching_threshold           When the size difference between current table and history table exceed this threshold, do not match history statistics. When value is 0.0, only match history statistics when the size of the two          ``hbo.history-matching-threshold``
+                                                            are exactly the same
```

Comment:
```suggestion
                                                            are exactly the same.
```
A very tiny nit, for punctuation.

Reply:
Thanks for the suggestion. I find the other session property descriptions all do not have the last punctuation, so add it for  all of them. Please take a look when available.